src/Chapter1.hs

```diff
@@ -653,7 +670,10 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n
+  | n `div` 10 == 0 = n
+  | otherwise = firstDigit(n `div` 10)
```

Comment:
In Haskell, function is usually separated from the applied arguments with spaces 👾 

```suggestion
  | otherwise = firstDigit (n `div` 10)
```